cartridges/int_algolia/cartridge/scripts/algolia/steps/algoliaProductDeltaIndex.js

```diff
@@ -132,9 +133,13 @@ exports.beforeStep = function(parameters, stepExecution) {
         }
     });
 
-    /* --- non-localized attributes --- */
+    /* --- non-localized/shared attributes --- */
     Object.keys(algoliaProductConfig.attributeConfig_v2).forEach(function(attributeName) {
-        if (!algoliaProductConfig.attributeConfig_v2[attributeName].localized) {
+        if (algoliaProductConfig.attributeConfig_v2[attributeName].computedFromBaseProduct) {
+            if (attributesToSend.indexOf(attributeName) >= 0) {
```

Comment:
we may merge these 2 `if` conditions.

Reply:
Thanks, your comment made me realize that the logic has become too complex.
I've merged the two loops into a single one. Let me know if it's better

I've removed the checks for duplicates it simplify it even more (`if (variantAttributes.indexOf(attribute) < 0)`)